packages/react-button/src/buttonBaseProps.ts

```diff
@@ -0,0 +1,21 @@
+import { ButtonProps } from './Button';
```

Comment:
lets add `.stories` suffix as well to be explicit that this belongs to storybook

Reply:
Done!